coralproject/talk
# [CORL-825] Allow sending moderation feedback to perspective

If there is a `don't merge` label on this, it's because we haven't merged in `CORL-688` yet and this branch is based off of it.

## What does this PR do?

Adds a configuration option to send approved and rejected comment moderation actions to perspective to help improve the tracking model.

## How do I test this PR?

- Head to Admin > Config > Moderation > Toxic Comment Filter
- Enable/allow the `Allow Coral to send moderation actions to Google` option
- Create a comment with a commenter account
- Approve/Reject it
- It'll get sent to Perspective, feel free to check logs and debug into `src/core/server/services/perspective/index.ts` functions
